src/components/Footer.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+	</head>
+	<body>
+		<!-------------------------Footer Section------------------------->
+
+		<footer class="col-12">
+			<div class="grid-col-4 col-12">
+				<div class="footer-container">
+					<div class="section-one">
+						<span
+							><img src="" alt="" />
+							<h5>
+								Always here to help you
+							</h5>
+						</span>
+
+						<div class="intro-and-links">
+							<h5>
+								All Rights are reserved
+							</h5>
+							<h5>
+								Made with
+								<i
+									class="material-icons footerIcon"
+									>favorites</i
+								>
+								by Mayank Kumar
+							</h5>
+						</div>
+					</div>
+				</div>
+
+				<div class="section-three">
+					<h5>Account</h5>
+					<h6><a href="">My Account</a></h6>
+					<h6>
+						<a href="">Privacy Policy</a>
+					</h6>
+				</div>
+
+				<div class="section-four">
+					<h5>ABOUT</h5>
+					<h6><a href="">Contact Us</a></h6>
+					<h6><a href="">About Us</a></h6>
+				</div>
+
+				<div class="section-six">
+					<p class="footer-address">
+						Address: Dwarka, New Delhi-110078
+					</p>
+					<p class="footer-address">
+						<span>Call US:</span>
+						1800 000 0000
+					</p>
+					<p class="footer-address">
+						<span>Email</span>
+						mayankkumark28@gmail.com
```

Comment:
can wrap it in a tag and provide href="mailto:emailId" to it. Through this your user can mail directly on a click

Reply:
Okay, I'll do that.